openapi3/templates/ApiClient.mustache

```diff
@@ -499,18 +522,51 @@ namespace {{packageName}}.Client
                 client.ClientCertificates = configuration.ClientCertificates;
             }
 
-            return client;
+            return client;*/
```

Comment:
Is it desirable to have the code generator generate a section of commented code like this?

Reply:
Good catch, I'll clean up.